### PR TITLE
Fix file path when checking application logs

### PR DESCRIPTION
**Pull request by ewolinetz**
Addressing #198 

May also do some other improvements within test script

### CODE REVIEW

**Comment by ewolinetz** [comment]
[test]

**Comment by ewolinetz** [comment]
[test]

**Comment by richm** [comment]
[test]

**Comment by ewolinetz** [comment]
[test]

**Comment by ewolinetz** [comment]
[test]

File: deployer/scripts/upgrade.sh [hunk 1]
```diff
@@ -558,12 +558,20 @@ function regenerate_config_and_support_objects() {
 
 function remove_triggers_and_IS() {
 
+  patchPIDs=()
   # This is getting only the names of DCs that still have a trigger defined
   # ?(@.spec.triggers[*].type) only matches on DC that have a value for 'type' under .spec.triggers[*]
   for dc in $(oc get dc -l logging-infra -o jsonpath='{.items[?(@.spec.triggers[*].type)].metadata.name}'); do
+    currentVersion=$(oc get $dc -o jsonpath='{.status.latestVersion}')
```
Comment (richm) [inline deployer/scripts/upgrade.sh]: nack - this should go into a separate commit

Comment (ewolinetz) [inline deployer/scripts/upgrade.sh]: bad habit, addressed

Comment (richm) [inline deployer/scripts/upgrade.sh]: oc get dc/$dc ...

**Comment by richm** [comment]
[test]

**Comment by ewolinetz** [comment]
[test]

File: deployer/scripts/upgrade.sh [hunk 2]
```diff
@@ -558,12 +558,20 @@ function regenerate_config_and_support_objects() {
...
 
+  patchPIDs=()
   # This is getting only the names of DCs that still have a trigger defined
   # ?(@.spec.triggers[*].type) only matches on DC that have a value for 'type' under .spec.triggers[*]
   for dc in $(oc get dc -l logging-infra -o jsonpath='{.items[?(@.spec.triggers[*].type)].metadata.name}'); do
+    currentVersion=$(oc get dc/$dc -o jsonpath='{.status.latestVersion}')
+
     oc patch dc/$dc -p='{"spec": { "triggers" : [] } }'
+
+    waitForChange $currentVersion $dc &
```
Comment (richm) [inline deployer/scripts/upgrade.sh]: need `dc/$dc` here too.

**Comment by richm** [comment]
[test]

**Comment by ewolinetz** [comment]
it's kicking off a deployment when patching for the image version change. looking at the deployer logs closer i see that the `oc deploy ... --latest` in update images is seeing the same message about a current deployment.

**Comment by ewolinetz** [comment]
[test]

**Comment by ewolinetz** [comment]
[test]

**Comment by ewolinetz** [comment]
[test]

**Comment by ewolinetz** [comment]
looks like `kube-system` is a project now; we should add it to the `operations` project list

**Comment by ewolinetz** [comment]
[test]

**Comment by ewolinetz** [comment]
[test]

**Comment by ewolinetz** [comment]
@sosiouxme @richm PTAL

**Comment by richm** [comment]
not sure why check-logs can't find all of the log messages, but LGTM

**Comment by sosiouxme** [comment]
LGTM

**Comment by ewolinetz** [comment]
[merge]

**Comment by ewolinetz** [comment]
[merge]
